src/preconnect.js

```diff
@@ -135,6 +145,20 @@ class Preconnect {
     return false;
   }
 
+  /** @private */
+  isPreloadSupported_() {
+    const tokenList = document.createElement('link').relList;
+    if (!tokenList || !tokenList.supports) {
+      return false;
+    }
+    try {
+      return tokenList.supports('preload');
```

Comment:
Ok so it seems this won't fail for `relList` and it will only fail for `classList` - which seems to implement the [DOMTokenList](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList) interface. 

Since we're using this only to check for `link``relList` then we should be good without try-catch. Dropping it.